app/serializers/provider_serializer.rb

```diff
@@ -88,6 +88,14 @@ def recruitment_cycle
     object.recruitment_cycle.year
   end
 
+  def created_at
+    object.changed_at.iso8601
```

Comment:
Shouldn't this be `created_at`?

Reply:
Seems like the tests should ensure that these are two different timestamps to test thoroughly.

Also, did you check to see if it just Does The Right Thing if you don't even bother with these methods?

Reply:
"Fri, 30 Aug 2019 16:19:35 UTC +00:00"
vs
"2019-08-30T16:26:33Z" (iso)